README.md

```diff
@@ -70,6 +70,17 @@ Please note that the date shown is the date that the change was released to the
 
 ### 16 April 2024
 
+The following changes are now available in the Sandbox:
+
+#### self-employment-business-api
+
+- Added 2 new optional fields transitionProfitAmount, transitionProfitAccelerationAmount in annualAdjustmentsType object for `v3.0` `Retrieve a Self-Employment Annual Submission and Create` and `Amend Self-Employment Annual Submission` endpoints.
```

Comment:
Should be:
`Retrieve a Self-Employment Annual Submission` and `Create and `Amend Self-Employment Annual Submission`